app/backend/wells/serializers.py

```diff
@@ -245,6 +245,7 @@ class Meta:
             "observation_well_number",
             "observation_well_status",
             "ems",
+            "aquifer",
```

Comment:
We're all over the show with quotes - could we at least be consistent in the same class. Maybe part of a bigger discussion, but I'm partial to single quotes in general.

https://www.python.org/dev/peps/pep-0008/#string-quotes

Reply:
@Sybrand I'm violently opposed to single quotes and think they're the devil.